src/js/helpers/JSONPUtil.js

```diff
@@ -0,0 +1,42 @@
+const JSONPUtil = {
+  /**
+   * Request JSONP data
+   *
+   * @todo: write test to verify this util is working properly
+   *
+   * @param {string} url
+   * @returns {Promise} promise
+   */
```

Comment:
@orlandohohmeier 
Also, we have this convention:
For every TODO we need a regarding issue on Github.
Then it would be like this:
```// TODO: #82301 - write test to verify this util is working properly```

This prevents us to have useless and forgotten todo-comments all around our code.